tests/test_script_alternative_type_checking/script.py

```
@@ -0,0 +1,6 @@
+from snakemake.iocontainers import Snakemake
+
+snakemake: Snakemake
+
+with open(snakemake.output[0], "w") as f:
```

Ruff (0.15.12)

[error] Undefined name `snakemake`

(F821)